web/portal/src/main/webapp/WEB-INF/conf/jdbc/configuration.xml

```diff
@@ -220,4 +220,19 @@
       <type>org.exoplatform.portal.mop.storage.listener.NodeDescriptionListener</type>
     </component-plugin>    
   </external-component-plugins>
+  <external-component-plugins>
+    <target-component>org.exoplatform.commons.file.services.NameSpaceService</target-component>
```

Comment:
where it is used ?

Reply:
here for the save of the file :
https://github.com/Meeds-io/gatein-portal/pull/661/files#diff-03d0a49739ad35956fd2c4da054ba21c373e8abccee15bffc510d008b4a31254R398